lib/till.rb

```diff
@@ -0,0 +1,84 @@
+require 'json'
+require_relative 'receipt'
+
+class Till
+  attr_reader :total, :tax, :order_items,  :price_list
+
+  TAXRATE = 8.64
+
+  def initialize
+    @business = ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.